.github/workflows/ci.yml

```diff
@@ -315,7 +320,17 @@ jobs:
       - name: Run smoke-test
         timeout-minutes: 5
         run: |
-          make RELEASE=true UNINSTALL=noclean smoke-test-tls-root
+          FLV_CMD=true FLV_SOCKET_WAIT=600 make RELEASE=true UNINSTALL=noclean smoke-test-tls-root
+      - name: Build examples
+        run: |
+          cargo build --release --bin produce
```

Comment:
Can we build this a single step? Every time, we add another example, we don't want to change the CI.